src/providers/export/vottJson.ts

```diff
@@ -32,7 +32,7 @@ export class VottJsonExportProvider extends ExportProvider<IVottJsonExportOption
         await results.forEachAsync(async (assetMetadata) => {
             return new Promise<void>(async (resolve) => {
                 const blob = await HtmlFileReader.getAssetBlob(assetMetadata.asset);
-                const assetFilePath = `vott-json-export/${assetMetadata.asset.id}.jpg`;
+                const assetFilePath = `vott-json-export/${assetMetadata.asset.name}`;
```

Comment:
Normalized VoTT json to follow same convensions as other export providers that same image binaries.